aggregator/src/datastore.rs

```diff
@@ -3622,6 +3705,29 @@ pub mod models {
             }
         }
 
+        /// Decode the fields of a client report using the supplied VDAF and construct a
+        /// [`LeaderStoredReport`].
+        pub(super) fn from_encoded_client_report(
```

Comment:
If this sticks around, it should be an `impl From<EncodedClientReport> for LeaderStoredReport`.